.idea/codeStyleSettings.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@pwalsh strange, I have added .idea folder to gitignore, but still see it in commit :)

Removed from GIT repo